### PR TITLE
Update README to add demo and stats

### PR DESCRIPTION
**Pull request by m2rads**
### Added
- Demo video
- Stars
- npm version
- Monthly downloads

### CODE REVIEW

**Comment by m2rads** [comment]
So seems like adding the demo to Github assets is not working. I think we should upload the video to the repo.

**Comment by m2rads** [comment]
Fixed

File: README.md [hunk 1]
```diff
@@ -1,6 +1,13 @@
 # Shortest
 **[shortest.com](https://shortest.com)**
 
+![GitHub Repo stars](https://img.shields.io/github/stars/anti-work/shortest?style=social)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[![npm version](https://img.shields.io/npm/v/@antiwork/shortest)](https://www.npmjs.com/package/@antiwork/shortest)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[![Downloads](https://img.shields.io/npm/dm/@antiwork/shortest)](https://www.npmjs.com/package/@antiwork/shortest)
```
Comment (slavingia) [inline README.md:4]: Personally think we should copy Firecrawl's approach, keeping the stars separate and having a CTA to like. I don't think including the stars by itself helps as it's already above. Wdyt?

Comment (m2rads) [reply README.md:4]: Perhaps we could add the Logo at the top and show the link to the website like how Firecrawl does. Just double checking would you like to add the rest of the stuff like Firecrawl or the Logo and the demo video is enough for now? 

image

Comment (slavingia) [reply README.md:4]: We could add the logo as it's already in the repo. Nextjs does similarly

File: README.md [hunk 2]
```diff
@@ -1,7 +1,19 @@
-# Shortest
-**[shortest.com](https://shortest.com)**
 
-AI-powered natural language end-to-end testing framework.
+<p align="center">
+  <img src="https://github.com/user-attachments/assets/0adf7c5d-296f-46a5-8071-57c408f02d43" alt="Shortest Logo" width="200" />
```
Comment (slavingia) [inline README.md:3]: I think we should use an SVG to mirror the app, or it doesn't work? Also, should be orange, the hex code equivalent of `text-orange-500`

**Pull request merged**